flexget/plugins/api_tmdb.py

```diff
@@ -269,6 +279,132 @@ def lookup(title=None, year=None, tmdb_id=None, imdb_id=None, smart_match=None,
+        if acc:
+            return acc.session_id
+        elif called_from_cli:
+            log.debug('Attempting to authorize FlexGet for account %s.', account)
+            session_id = authenticate(account)
+            acc = TmdbUserAuth(account, session_id)
+            session.add(acc)
+            return session_id
+        else:
+            raise plugin.PluginError('Account %s has not been authorized. See `flexget tmdb auth -h` on how to.' %
+                                     account)
+
+
+def do_cli(manager, options):
+    if options.action == 'auth':
+        if not (options.account):
+            console('You must specify an account (local identifier) so we know where to save your access token!')
+            return
+        try:
+            get_session_id(options.account, called_from_cli=True)
+            console('Successfully authorized Flexget app on themoviedb.org. Enjoy!')
+            return
+        except plugin.PluginError as e:
+            console('Authorization failed: %s' % e)
+    elif options.action == 'show':
+        with Session() as session:
+            if not options.account:
+                # Print all accounts
+                accounts = session.query(TmdbUserAuth).all()
+                if not accounts:
+                    console('No tmdb authorizations stored in database.')
+                    return
+                console('{:-^21}|{:-^40}|{:-^28}'.format('Account', 'Session Id', 'Created'))
+                for auth in accounts:
+                    console('{:<21}|{:>40}|{:>28}'.format(
+                        auth.account, auth.session_id, auth.created.strftime('%Y-%m-%d')))
+                return
+            # Show a specific account
+            acc = session.query(TmdbUserAuth).filter(TmdbUserAuth.account == options.account).first()
+            if acc:
+                console('Session Id is %s' % acc.session_id)
+            else:
+                console('Flexget has not been authorized to access your account.')
+    elif options.action == 'delete':
+        if not options.account:
+            console('Please specify an account')
+            return
+        try:
+            delete_account(options.account)
+            console('Successfully deleted your access token.')
+            return
+        except plugin.PluginError as e:
+            console('Deletion failed: %s' % e)
+
+
+@event('options.register')
+def register_parser_arguments():
+    acc_text = 'local identifier which should be used in your config to refer these credentials'
+    # Register subcommand
+    parser = options.register_command('tmdb', do_cli, help='view and manage tmdb authentication.')
+    # Set up our subparsers
+    subparsers = parser.add_subparsers(title='actions', metavar='<action>', dest='action')
+    auth_parser = subparsers.add_parser('auth', help='authorize Flexget to access your themoviedb.org account')
+
+    auth_parser.add_argument('account', metavar='<account>', help=acc_text)
```

Comment:
> Also, if that argument is required, add the required=True param to its creating, instead of later checking it in code.

It's not required for the `show` parser. So if I switch to using parent parser I can't use that.

I could add a new command `list` which does what `show` currently does when no `account` is given.